packages/block-library/src/cover/edit.js

```diff
@@ -379,11 +379,13 @@ function CoverEdit( {
 					}
 					isDisabled={ ! hasBackground }
 				/>
-				<FullHeightAlignment
+				<FullHeightAlignmentControl
 					isActive={ isMinFullHeight }
 					onToggle={ toggleMinFullHeight }
 					isDisabled={ ! hasBackground }
 				/>
+			</BlockControls>
+			<BlockControls group="other">
```

Comment:
We could also do the same in the native version for its toolbar buttons:
https://github.com/WordPress/gutenberg/blob/bac8b7322d1c56ce51c185fd6364e04585a8d425/packages/block-library/src/cover/edit.native.js#L268-L276
```
		<BlockControls group="other">
			<ToolbarButton
				title={ __( 'Edit cover media' ) }
				icon={ replace }
				onClick={ open }
			/>
		</BlockControls>
```

Would you mind if I push this change?

Reply:
Yes, it perfectly fine to align 👍🏻